eripheebs/cloudKitTracker
# Issue 12 click pin to show user info

#### What does this PR do?
Clicking a pin now shows the username and the last update time of the location

#### What are the relevant tickets?
Issue 12

#### Where should the reviewer start?
ViewController.swift, usermodel.swift

#### How should this be manually tested?
Load up the app, check that clicking a pin results in the correct information being shown
